src/comp/pages/CreateProjectPage.tsx

```diff
@@ -99,14 +247,103 @@ class CreateProjectPage extends React.Component<
                 helperText={introductionHelperText}
               />
             </div>
+            <div className={CommonStyles.createProjectContentsBox}>
+              <div className={CommonStyles.content_subtitle}>タグ</div>
+              <ScrollableTagList
+                tagIDs={this.state.tagIDs}
+                onTagClick={this.handleTagListClick}
+              />
+              <TextField
+                id="add_tag_name"
+                label="追加するタグの名前"
+                margin="normal"
+                multiline
+                variant="outlined"
+                fullWidth
+                onChange={this.handleTagNameInputChange}
+                error={this.state.tagIDErrorCode !== 0}
+                value={this.state.addingTagName}
+                helperText={tagIDHelperText}
+              />
+              <Box m={1} />
+              <CreateSendButton
+                canSend={
+                  checkBlankSpace(this.state.addingTagName) &&
+                  this.state.tagIDErrorCode === 0
+                }
+                handleSendButton={this.handleAddTagClick}
+                outline
+              >
+                追加
+              </CreateSendButton>
+            </div>
+            <Box m={2} />
+            <div className={CommonStyles.createProjectContentsBox}>
+              <div className={CommonStyles.content_subtitle}>関連サイト</div>
+              <List style={{ padding: 0 }}>
+                {this.state.sites.map((site, index) => (
+                  <ListItem key={index}>
+                    <ListItemText primary={site.title} secondary={site.url} />
+                    <ListItemSecondaryAction>
+                      <IconButton
+                        onClick={() => {
+                          this.handleSiteRemoveButton(index);
+                        }}
+                      >
+                        <CloseIcon />
+                      </IconButton>
+                    </ListItemSecondaryAction>
+                  </ListItem>
+                ))}
```

Comment:
```suggestion
                {this.state.sites.map((site, index) => (
                  <Card variant="outlined" key={index}>
                    <ListItem>
                      <ListItemText primary={site.title} secondary={site.url} />
                      <ListItemSecondaryAction>
                        <IconButton
                          onClick={() => {
                            this.handleSiteRemoveButton(index);
                          }}
                        >
                          <CloseIcon />
                        </IconButton>
                      </ListItemSecondaryAction>
                    </ListItem>
                  </Card>
                ))}
```
- [Outlined Card](https://material-ui.com/ja/components/cards/#OutlinedCard.tsx)
デザインを合わせましょう